HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # Version History
 
+## 1.0.23 / 2021-06-29
+
+- Add certificate validation to batch calls
```

Comment:
this is almost not worth mentioning, but i figured you're the one that should have the option to decline it. But technically this doesnt add cert validation, it adds the option of cert validation.

Reply:
Updated history to make it clearer that certificate validation is optional